<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Результат:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_11">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_5">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_12">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_7">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_8">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_9">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="pushButton_14">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>×</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pushButton_10">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButton_15">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>del</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pushButton_16">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="pushButton_13">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>÷</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
